topdown/topdown_test.go

```diff
@@ -1446,6 +1449,535 @@ func TestTopDownURLBuiltins(t *testing.T) {
 	}
 }
 
+func TestTopDownJWTEncodeSignPayloadErrors(t *testing.T) {
```

Comment:
Any chance we can move these into `tokens_test.go` instead of the monolithic `topdown_test.go`?

Reply:
If you do not mind I prefer to do that later. Every time I move things around it generates more requests for change and more comments in a never ending cycle.